src/model/account/PublicAccount.ts

```diff
@@ -101,7 +101,7 @@ export class PublicAccount {
     /**
      * Create Builder object
      */
-    toBuilder(): KeyDto {
-        return new KeyDto(Convert.hexToUint8(this.publicKey));
+    toBuilder(): PublicKeyDto {
```

Comment:
isn't this a breaking change to clients?

Reply:
Yes, those methods should have been `private / @internal` in this SDK I guess.  It's a secondary effect of the schema name changes.

Reply:
so, expectation is that no one should be calling this publicly?

Reply:
I would be really really surprised.